packages/web3-eth-contract/types/index.d.ts

```diff
@@ -95,10 +95,16 @@ export interface EstimateGasOptions {
 }
 
 export interface ContractOptions {
```

Comment:
The ``ContractOptions`` interface can inherit from the ``Web3ModuleOptions`` interface in the ``web3-core`` module. :)

Reply:
Cool, didn’t know. Fixed in https://github.com/ethereum/web3.js/pull/2939/commits/0a0b2e7b6cf8e697c83f942307af6c995c3b4ae2.